log4j-layout-template-json/src/main/java/org/apache/logging/log4j/layout/template/json/resolver/MessageResolver.java

```diff
@@ -134,7 +134,10 @@ private static void resolveString(
             jsonWriter.writeObjectStart();
             jsonWriter.writeObjectKey(fallbackKey);
         }
-        if (message instanceof StringBuilderFormattable) {
+        if (message instanceof CharSequence) {
+            final CharSequence sequence = (CharSequence) message;
+            jsonWriter.writeString(sequence);
```

Comment:
Is the local variable necessary?

Reply:
Nope, I just find it easier to read and wrap my mind around when there is one operation per line.

Reply:
I agree with @ppkarwasz, this just makes the code more verbose than it has to be IMO.
Type casting is not an "operation", it does not do anything.

Reply:
Fixed in 5a37bafb.